.github/workflows/lint.yml

```diff
@@ -46,6 +46,7 @@ jobs:
         if: env.GIT_DIFF
         uses: actions-rs/tarpaulin@v0.1.3
         with:
+          version: 0.22.0
```

Comment:
`action-rs/tarpaulin` now has the issue on coverage report. The latest available version is 0.22.0
https://github.com/actions-rs/tarpaulin/issues/15#issuecomment-1413029297